nexus/usecases/intelligences/update.py

```diff
@@ -45,3 +49,24 @@ def update_contentbase(
         contentbase.save()
 
         return contentbase
+
+
+class UpdateContentBaseTextUseCase():
+
+    def __init__(self):
+        pass
+
+    def update_contentbasetext(
+            self,
+            contentbasetext_uuid: str,
+            text: str = None,
+    ):
+
+        contentbasetext = get_by_contentbasetext_uuid(contentbasetext_uuid)
+
+        if text:
+            contentbasetext.text = text
+
+        contentbasetext.save()
```

Comment:
I recommend put the save in `if` scope and use the parameter update_fields

Reply:
Changed on the last commit. "Add: Improve save performace"